lms/djangoapps/teams/serializers.py

```diff
@@ -17,6 +18,10 @@ class UserMembershipSerializer(serializers.ModelSerializer):
 
     Used for listing team members.
     """
+    profile_configuration = deepcopy(settings.ACCOUNT_VISIBILITY_CONFIGURATION)
+    profile_configuration['shareable_fields'].append('url')
```

Comment:
It was my understanding that the url is included in the expanded case so that there is a link back to the original reference, since this object does not originate from that endpoint.  @andy-armstrong is that right?